src/App.vue

```diff
@@ -1,11 +1,22 @@
 <template>
   <div class="wrapper">
+    <Navbar v-show="showNavbar" />
     <router-view></router-view>
   </div>
 </template>
 
 <script>
+import Navbar from "./components/Navbar";
+
 export default {
   name: "App",
+  components: { Navbar },
+  computed: {
+    showNavbar() {
```

Comment:
Verovatno je moguce da imas 2 view-a, Auth i ovaj kada nije ulogovan,
I onda da na njima kada radis ucitavanje childrena imas router-view i tamo onda Navbar samo za logged in
Ali ok, ne moras to sada. Samo bih promenio naziv `showNabar` u `isNavbarVisible`

Ali mozda i mozes samo da proveris da li je ulogovan umesto ruta?